### DISCUSSION TITLE
Alpine property not defined after livewire loop item order change (documentation update)

### DISCUSSION THREAD

**Discussion by mattyforth** [livewire/livewire#7785] [7. Bugs (archived)]
### Livewire version

3.3.5

### Laravel version

10.41.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Open the Wirebox URL.
2. Open up the inspector console.
3. Mark the top todo item as complete, it should move to the bottom.
4. Now mark the todo that is now at the top, also as complete.
5. View console error that `expanded is not defined`.

### Repository or Wirebox URL

https://wirebox.app/b/ywde0

### Additional information

I'm not 100% sure if this is a bug or not, but I feel that the documentation should be updated if this is the intended behaviour, as it was my understanding that adding `wire:key` would prevent this issue. I couldn't find any extra information surrounding a possible issue when using `x-data` inside a regular blade loop.

## The fix
To solve this problem, you can add `wire:ignore` to the parent div inside the `$todos` loop, this obviously will introduce some different side effects, but it does appear to fix the original problem.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
I think looks like a bug. Any chance you could PR a failing test?

**Reply by mattyforth** [reply]
Yes of course, I will sort that out.

**Reply by mattyforth** [reply]
HI @PhiloNL, see pull request here: #7850

As said in the PR, not sure if I've put things where they should be, but happy to tweak anything if needed!

**Reply by PhiloNL** [reply]
Perfect! Thanks for submitting :)
